core/src/validator.rs

```diff
@@ -2535,23 +2540,28 @@ mod tests {
     #[test]
     fn test_interval_check() {
         fn new_snapshot_config(
-            full_snapshot_archive_interval_slots: Slot,
-            incremental_snapshot_archive_interval_slots: Slot,
+            full_snapshot_archive_interval_slots: NonZeroU64,
+            incremental_snapshot_archive_interval_slots: NonZeroU64,
         ) -> SnapshotConfig {
             SnapshotConfig {
                 full_snapshot_archive_interval_slots,
                 incremental_snapshot_archive_interval_slots,
                 ..SnapshotConfig::default()
             }
         }
-
-        assert!(is_snapshot_config_valid(
-            &new_snapshot_config(300, 200),
-            100
-        ));
+        macro_rules! nz {
```

Comment:
Yeah. As context, the original motivation for the `nz!` macro to start was that I found it very invasive to have to do `NonZeroU64::new(X).unwrap()` instead of `X` in these test cases. That is why I went with minimal length macro name, and in very narrow scope (as opposed to a more properly named macro at higher level)